updatecli/values.d/scm.yaml

```
@@ -1,8 +1,29 @@
-scm:
-  enabled: true
-  default:
+scms:
+  main:
+    enabled: true
     user: 'fleet-bot'
     email: 'fleet@suse.de'
     owner: rancher
     repository: fleet
     branch: main
+  v12:
+    enabled: true
+    user: 'fleet-bot'
+    email: 'fleet@suse.de'
+    owner: rancher
+    repository: fleet
+    branch: 'release/v0.12'
+  v11:
+    enabled: true
+    user: 'fleet-bot'
+    email: 'fleet@suse.de'
+    owner: rancher
+    repository: fleet
+    branch: 'release/v0.11'
+  v10:
+    enabled: true
+    user: 'fleet-bot'
+    email: 'fleet@suse.de'
+    owner: rancher
+    repository: fleet
+    branch: 'release/v0.10'
```

You could use a YAML anchors to avoid repeating these (provided that updatecli's YAML parser supports it):
```suggestion
scms:
  main: &main
    enabled: true
    user: 'fleet-bot'
    email: 'fleet@suse.de'
    owner: rancher
    repository: fleet
    branch: main
  v12:
    <<: *main
    branch: 'release/v0.12'
  v11:
    <<: *main
    branch: 'release/v0.11'
  v10:
    <<: *main
    branch: 'release/v0.10'
```

---

Nice! I've tried this, and a local `updatecli diff` run suggests that updatecli supports it :rocket: